website/next.config.js

```diff
@@ -47,6 +47,11 @@ export default withNextra({
       destination: '/en/',
       permanent: false,
     },
+    {
+      source: '/en/substreams/',
+      destination: '/en/substreams/README/',
+      permanent: false,
+    },
```

Comment:
This is so that switching from another language's `/substreams` page to English works.